optimum/habana/transformers/trainer.py

```diff
@@ -1858,7 +1858,7 @@ def prediction_loop(
             if not self.already_wrapped_for_hpu_graphs:
                 from habana_frameworks.torch.hpu import wrap_in_hpu_graph
 
-                model = wrap_in_hpu_graph(model)
+                model = wrap_in_hpu_graph(model, disable_tensor_cache=args.disable_tensor_cache_hpu_graphs, max_graphs=args.max_hpu_graphs)
```

Comment:
Well, it's actually a comment from Transformers so that means that this part of the code will be removed in Transformers v5 (which we don't know when it will be released, there is no ETA). In the meantime, we still have to maintain it so let's keep the arguments here.